src/style.css

```diff
@@ -1,3 +1,112 @@
-body {
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #060b35;
+}
+
+* {
   margin: 0;
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__name {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 1.5;
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__star-rating {
+  margin-bottom: 24px;
+}
+
+.card__price,
+.card__star-rating {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.card__price {
+  padding-bottom: 16px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-cost {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+}
+
+.card__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 166px;
+  height: 40px;
+  background-color: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  color: #fff;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.card__button:hover {
+  background-color: #fff;
+  color: #00acdc;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  margin-right: 4px;
```

Comment:
last star shouldn't have margin

Reply:
How can I do it?

Reply:
Hey, you can remove margin-right for last star using pseudo elements like :nth-child and :not.

Anyways If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your questions in GitHub will be visible to mentors only after you re-request review